public/stylesheets/main.css

```diff
@@ -45,6 +45,21 @@ div.player {
   border-radius: 5px;
 }
 
+div.it { 
```

Comment:
We could refactor these so there aren't repeated properties. Then we can assign multiple classes to divs. (I think that will work)